lib/tasks/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
why add this? do we still need this folder

Reply:
Really just putting this file back (originally removed back in October) so that both `lib/assets` and `lib/tasks` remain in the repo (which is the default for new Rails apps).